cirq-core/cirq/linalg/tolerance.py

```diff
@@ -33,7 +33,7 @@ def all_near_zero(a: 'ArrayLike', *, atol: float = 1e-8) -> bool:
 
 
 def all_near_zero_mod(
-    a: Union[float, complex, Iterable[float], np.ndarray], period: float, *, atol: float = 1e-8
+    a: Union[complex, Iterable[float], np.ndarray], period: float, *, atol: float = 1e-8
```

Comment:
```suggestion
    a: Union[complex, Iterable[complex], np.ndarray], period: float, *, atol: float = 1e-8
```

Reply:
I actually went the opposite direction. The function fails on complex numbers (there's a mod operation in it), so I'm not sure why complex was an option at all.